### PR TITLE
feat : wishlist and cart pages added

### PR DESCRIPTION
**Pull request by yashodaAgarwal**
Please review my wishlist and cart pages

### CODE REVIEW

File: Styles/var.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+*{
+    font-size: 1rem;
+    line-height: 1.3rem;
+}
```
Comment (Debarshi95) [inline Styles/var.css:5]: Used * selector. These styles will be applied to all selectors. Are you sure about this?

**Review by Debarshi95 — APPROVED** [review]
Looks good. Can update the mentioned issue in next PR.

File: Styles/var.css (hunk 1 above)
Comment (yashodaAgarwal) [reply Styles/var.css:5]: Yes This is the basic size of all my text nd if I need any specific size I have changed accordingly

Comment (Debarshi95) [reply Styles/var.css:5]: I have approved the PR. You can merge if you're sure about this.

File: pages/wishlist.html [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+        </button>
+        <button class="badge-button badge-lg"
+        onClick="window.location.href='./Cart.html';"
+        >
+          <i class="fas fa-shopping-cart fa-2x"></i>
+          <span class="icon-button-badge flex center icon-xl">16</span>
+        </button>
+      </div>
+    </nav>
+    <h1 class="mywishlist-heading center">My Wishlist</h1>
```
Comment (ArpitPatni) [inline pages/wishlist.html:51]: Please use margin top with my wishlist heading.

File: pages/Cart.html [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        </button>
+        <button class="badge-button badge-lg"
+        onClick="window.location.href='./Cart.html';"
+        >
+          <i class="fas fa-shopping-cart fa-2x"></i>
+          <span class="icon-button-badge flex center icon-xl">16</span>
+        </button>
+      </div>
+    </nav>
+<h1 class="mycart-heading">My Cart</h1>
```
Comment (ArpitPatni) [inline pages/Cart.html:49]: Use here also margin top ,always give a certain gap between navbar and heading.

**Pull request merged**